styles/component-style/image.css

```diff
@@ -0,0 +1,41 @@
+.responsive-img-container{
+    width:100%;
+}
+
+.responsive-img {
+   width:70%; 
```

Comment:
width with 100% will give better result.

Reply:
+1  Give width 100% to the image instead of the container.